README.md

```diff
@@ -111,7 +111,10 @@ Providing options are optional, entities can be listed directly, see example bel
 | unit | string | optional | Set a custom unit of measurement, overrides `unit` set in base config.
 | show_state | boolean | optional | Display the current state.
 | show_indicator | boolean | optional | Display a color indicator next to the state, (only when more than two states are visible).
+| show_line | boolean | optional | Set to false to turn off the line. Does not impact the fill.
+| show_points | boolean | optional | Set to false to turn off the points.
 | state_adaptive_color | boolean | optional | Make the color of the state adapt to the entity color.
+| y_axis | string | optional | If 'secondary', displays using the secondary y-axis on the right.
```

Comment:
Could we make this a boolean instead? What do you think?

Reply:
I saw there was another feature request to have multiple x-axis scales (more than 2). So I wondered if people might want to do the same thing in the y-axis - multiple data series with independent scales. So you could name them "scale1", "scale2" etc. And then if they were called "primary" or "secondary" they would have the extra meaning as displaying the graph labels on the left or right. Not that I've coded that yet - so far anything other than "secondary" implies "primary". Do you think this would be worthwhile, or overkill?

Reply:
Probably overkill, would get pretty messy I would think.
But sure, let's keep it to future proof.